test/applications/MortgageApplication/application-conf/Cobol.properties

```diff
@@ -10,6 +10,11 @@ cobol_fileBuildRank=
 # Rules defined in application.properties
 cobol_resolutionRules=[${copybookRule}]
 
+#
+# COBOL dependencySearch configuration
+# searchPath defined in application.properties
+cobol_dependencySearch=${copybookSearch}
```

Comment:
I thought this was in previous versions of zAppBuild? Why is git showing it like this is a new addition to the file?